.github/pull_request_template.md

```diff
@@ -1,30 +1,20 @@
 ## Relevant Issues
-<!-- List relevant issue tickets here. -->
-<!-- Say "Closes #0" for issues that the PR resolves, replacing 0 with the issue number. -->
+<!--
+It is mandatory to link to an issue that has been approved by a Core Developer, indicated by an "approved" label.
+Issues can be skipped with explicit core dev approval, but you have to link the discussion.
+-->
 
-
-## Description
-<!-- Describe how you've implemented your changes. -->
-
-
-## Reasoning
-<!-- Outline the reasoning for how it's been implemented. -->
+<!-- Link the issue by typing: "Closes #<number>" (Closes #0 to close issue 0 for example). -->
 
 
-## Screenshots
-<!-- Remove this section if the changes don't impact anything user-facing. -->
-<!-- You can add images by just copy pasting them directly in the editor. -->
-
-
-## Additional Details
-<!-- Delete this section if not applicable. -->
-
+## Description
+<!-- Describe what changes you made, and how you've implemented them. -->
 
 ## Did you:
 <!-- These are required when contributing. -->
 <!-- Replace [ ] with [x] to mark items as done. -->
 
 - [ ] Join the [**Python Discord Community**](https://discord.gg/python)?
-- [ ] If dependencies have been added or updated, run `pipenv lock`?
-- [ ] **Lint your code** (`pipenv run lint`)?
-- [ ] Set the PR to **allow edits from contributors**?
+- [ ] Read all the comments in this tempelate?
```

Comment:
How will we be encouraging linting, if at all?

Reply:
It is mentioned inside the contrib guidelines. It may be worth to mention them here again but in the end the failed lint will be the same thing, we’ll end up with a second fix commit.

Reply:
Maybe we should link the contributing guidelines in a comment within this PR template?

Reply:
It is already mentioned line 20 but I feel like it could even be the first line of the template. We could also add a sentence about when this template should be used.

Reply:
I somehow completely missed that link with my eyes. Sorry about that, this PR looks good to me.